tools/js-test-runner/jest-config/default.config.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const glob = require( 'glob' );
+
+const getBabelConfig = () => {
+	// support babel.config.* | .babelrc | .bablerc.*
```

Comment:
I don't think we need to look for `.babelrc`. Where it exists Babel should already find it when it's needed.

At least that's what I get from https://babeljs.io/docs/en/config-files

Reply:
@anomiex But I need to check if it exists to apply the default one if I pass the default config for `configFile` it will use it instead local, not?

Reply:
Found a better reference. https://babeljs.io/docs/en/options#configfile says it's just for `babel.config.js` type files, and in particular says

> NOTE: This option does not affect loading of .babelrc.json files, so while it may be tempting to do configFile: "./foo/.babelrc.json", it is not recommended. If the given .babelrc.json is loaded via the standard file-relative logic, you'll end up loading the same config file twice, merging it with itself. If you are linking a specific config file, it is recommended to stick with a naming scheme that is independent of the "babelrc" name.

Reply:
@anomiex Thanks for pointing it. Removed `.babelrc` require into new commit